sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/MutationGroupEncoderTest.java

```diff
@@ -528,6 +528,32 @@ public void dateKeys() throws Exception {
     verifyEncodedOrdering(schema, "test", keys);
   }
 
+  @Test
+  // Test for Jira issue BEAM-4862
```

Comment:
I believe this comment becomes irrelevant once the issue is fixed. If we want to keep it, it would be better to put it into the code, because then it will comment on the reason why the code was implemented one way, not the other.

Reply:
+1